src/sentry/integrations/slack/webhooks/base.py

```diff
@@ -19,10 +19,19 @@
 NOT_LINKED_MESSAGE = "You do not have a linked identity to unlink."
 ALREADY_LINKED_MESSAGE = "You are already linked as `{username}`."
 
+import logging
+
+from sentry.utils import metrics
+
+logger = logging.getLogger(__name__)
+
 
 class SlackDMEndpoint(Endpoint, abc.ABC):
     slack_request_class = SlackDMRequest
 
+    _METRICS_SUCCESS_KEY = "slack.integrations.slack.dm_endpoint.success."
+    _METRIC_FAILURE_KEY = "slack.integrations.slack.dm_endpoint.failure."
```

Comment:
going by what we have for discord, i think these should be `sentry.integrations.slack.dm_endpoint.success `and `sentry.integrations.slack.dm_endpoint.failure`

```suggestion
    _METRICS_SUCCESS_KEY = "sentry.integrations.slack.dm_endpoint.success"
    _METRIC_FAILURE_KEY = "sentry.integrations.slack.dm_endpoint.failure"
```

Reply:
for success and failure, instead of emitting a new metric should we group all failures together and differentiate them by tags?

Reply:
e.g. `metrics.incr(self._METRICS_SUCCESS_KEY, tags={"action": "unlink_user"}, sample_rate=1.0)`

Reply:
these are just options, i don't think anything is blocking except the key naming with `sentry` in front

Reply:
ill update how do do metrics once i figure out which ones are worth keeping after collecting some data.

Reply:
ok